src/strict/processors/ollama_processor.py

```
@@ -31,17 +38,59 @@ async def process(self, request: ProcessingRequest) -> OutputSchema:
+                httpx.TimeoutException,
+                httpx.ConnectError,
+                json.JSONDecodeError,
+            ) as e:
+                duration = (time.time() - start_time) * 1000
+                return OutputSchema(
+                    result="",
+                    validation=ValidationResult(
+                        status=ValidationStatus.FAILURE,
+                        is_valid=False,
+                        input_hash="hash_placeholder",
+                        errors=(f"Ollama error: {str(e)}",),
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)